io/disk/port_bounce.py

```diff
@@ -294,6 +301,21 @@ def get_fc_host(self, path):
             if "host" in line:
                 return line
 
+    def get_nvmf_host(self, pci_addr):
+        '''
+        find and returns fc_host for given pci_address of nvmf adapter
+
+        :param pci_addr: pci_address of nvmf adapter, space separated.
+        :returns: fc_host name of given pci_address.
+        :rtype: string
+        '''
+        cmd = 'ls -l /sys/class/fc_host/'
+        out = process.getoutput(cmd)
+        for line in out.splitlines():
+            if '/%s/' % pci_addr in line:
+                return line.split('/')[-1]
+        return ""
```

Comment:
yes, Abdul but using pci_address, which was not used in earlier code.
Since I have added the pci_address parameter to yaml now for nvmf, I have changed it accordingly.
thanks for pointing this.